utils.js

```diff
@@ -1,21 +1,18 @@
 // baseline estimates, used to improve performance
 var TX_EMPTY_SIZE = 4 + 1 + 1 + 4
 var TX_INPUT_BASE = 32 + 4 + 1 + 4
-var TX_INPUT_PUBKEYHASH = 107
 var TX_OUTPUT_BASE = 8 + 1
-var TX_OUTPUT_PUBKEYHASH = 25
 
 function inputBytes (input) {
-  return TX_INPUT_BASE + (input.script ? input.script.length : TX_INPUT_PUBKEYHASH)
+  return TX_INPUT_BASE + input.script.length
```

Comment:
not clear what happens if the input provided has no script 🤔

Reply:
It should throw and error since it is a change to the [API.](https://github.com/bitcoinjs/coinselect/pull/22#issuecomment-322345043)